format_spec/fragment.md

```diff
@@ -126,7 +126,8 @@ For *var-sized dimensions*, the `1DRange` format is:
 
 ### Tile Offsets
 
-The tile offsets is a [generic tile](./generic_tile.md) with the following internal format:
+Tile offsets should be set _only_ on variable-sized tiles, to refer to each data tile's starting byte offset. \
```

Comment:
That is incorrect. The tile offsets are for any on disk tiles. They might be compressed so we cannot calculate where they are on disk.